data_validation/src/spark_rapids_validation_tool/validation_scripts/metadata_validation.py

```diff
@@ -136,7 +136,7 @@ def map_to_string(data):
     # if not specified any included_columns, then get all numeric cols and string and map cols
     excluded_columns_list = [e.strip() for e in exclude_column.split(",")]
     metrics_cols = [i.strip() for i in include_column.split(",") if i not in excluded_columns_list]
-    if i in ['None', 'all']:
+    if include_column in ['None', 'all']:
```

Comment:
The usage of `None here is ambiguous. Using a string value that is equal to the python keyword `None` will cause misinterpretation of the value.